OPT_mission.Tanoa/beam/functions/fnc_initaddbeamdialog.sqf

```diff
@@ -0,0 +1,47 @@
+/**
+* Author: form
+* initialize addBeam dialog with coordinates of actual beam positions
+*
+* Arguments:
+* None
+*
+* Return Value:
+* None
+*
+* Example:
+* [] call fnc_initAddBeamDialog.sqf;
+*
+*/
+#include "script_component.hpp"
+
+#define ADDBEAM_IDC 1413
```

Comment:
Dieses Makro wird genau so auch schon in `fnc_addbeam.sqf` definiert. Ich würde die Definition in ein include auslagern, sodass es nur an einer Stelle definiert wird (falls man das mal ändern muss)